content/07.acknowledgements.md

```diff
@@ -12,3 +12,8 @@ Access to the New York Times Annotated Corpus (NYTAC) is available upon request
 The authors would like to thank Ariel Hippen Anderson for evaluating potential missing preprint to published version links.
 We also would like to thank Richard Sever and the _bioRxiv_ team for their assistance with access to and support with questions about preprint full text downloaded from _bioRxiv_. 
 This work was supported by [Grant GBMF4552](https://www.moore.org/grant-detail?grantId=GBMF4552) from the Gordon Betty Moore Foundation and by NIH T32HG00046, Computational Genomics training grant, from the National Human Genome Reserach Institute (NHGRI).
+
+## Competing Interest
+
+The authors declare that they have no known competing financial interests or personal relationships that could have influenced the work reported in this paper.
```

Comment:
Marvin is likely to need to report his employer, since elsevier is a publisher (i.e., a company that works with literature/preprints). I'd recommend including a stub that reminds us so that we don't forget.
```suggestion
Need to note here.
```